Resources/views/Gallery/index.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{% sonata_template_box 'This is the gallery index template. Feel free to override it.' %}
+{% sonata_template_box 'This is the gallery index template. Feel free to override it.' 'SonataMediaBundle'%}
```

Comment:
I investigated the other approach would be to merge this PR to CoreBundle:

https://github.com/sonata-project/SonataCoreBundle/pull/402